client/src/components/TranslationInfoBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Table } from "semantic-ui-react";
+import { parseDate } from "./helpers/parseDate";
+
+const TranslationInfoBar = ({
+  startTranslationDate,
+  submitTranslationDate,
+  translatorName
+}) => {
+  const parsSubmitTranslationDate = parseDate(submitTranslationDate);
+  const parsStartTranslationDate = parseDate(startTranslationDate);
+
+  return (
+    <Table.Row>
+      <Table.HeaderCell textAlign="center" colSpan="3">
+        Translation started on {parsStartTranslationDate} and submitted on{" "}
```

Comment:
This looks like another autoformatting artifact `{" "}` cause by splitting this line? I think it can be removed.

Reply:
I removed it